saleor/core/utils/update_mutation_manager.py

```diff
@@ -0,0 +1,16 @@
+def get_editable_values_from_instance(instance):
+    field_names = [
+        field.name for field in instance._meta.model._meta.fields if field.editable
```

Comment:
Not much I can find on `field.editable` in Django docs, but from what I've digged into source that's a property of everything that inherits from `django.db.model.fields.Field` (relation fields included). Given that related fields have to be checked for changes in custom ways, does Django model consider such fields editable?

Reply:
1. Related fields that can be changed over the mutation but don't have direct representation on the base model will not be returned by the function (e.g., attributes in `ProductVariantUpdateInput`) and needs custom handler. However, `product_id` will be returned.
2. We already use `field.editable` in `BaseMutation.construct_instance`
3. If your concern is about omitting some field, take a look on tests, that should fail if don't check any input field

Does it answer your question?